src/main/java/de/hpi/bpt/argos/common/EventProcessingPlatformUpdaterImpl.java

```diff
@@ -176,4 +179,25 @@ public EventPlatformFeedback deleteEventQuery(EventQuery eventQuery) {
 
 		return new EventPlatformFeedbackImpl(request);
 	}
+
+	/**
+	 * This method waits until the eventProcessingPlatform is reachable.
+	 */
+	private void waitForEventProcessingPlatform() {
+		boolean reachable = RestRequestFactoryImpl.getInstance().isReachable(EventProcessingPlatformUpdater.getHost());
+		final long retryTime = 5000;
+		long startTime = System.currentTimeMillis();
+
+		while (!reachable) {
+			logger.info(String.format("waiting for event processing platform to be up ... (%1$d ms)", System.currentTimeMillis() - startTime));
+
+			try {
+				Thread.sleep(retryTime);
+			} catch (InterruptedException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS2142)